HW_1/task_1.py

```diff
@@ -0,0 +1,16 @@
+# Задача 2: Найдите сумму цифр трехзначного числа
+
+n = int(input("Введите трехзначное число n: "))
+
+if (99 < n < 1000):
+    a = n % 10
+    b = round((n-a)/10 % 10)
```

Comment:
Елена, вы пошли достаточно сложным путем. Есть способ попроще.
```suggestion
    b = n // 10 % 10
```